libraries/classes/Controllers/Database/SearchController.php

```diff
@@ -79,7 +79,7 @@ public function __invoke(ServerRequest $request): void
         }
 
         // Main search form has been submitted, get results
-        if (isset($_POST['submit_search'])) {
+        if (isset($request->getParsedBodyParam('submit_search'))) {
```

Comment:
```suggestion
        if ($request->getParsedBodyParam('submit_search') !== null) {
```
It's definitely set. Maybe check for `!== null`?